addons/test_mail_sms/tests/test_sms_composer.py

```diff
@@ -133,6 +133,22 @@ def test_composer_comment_nofield(self):
             sms_composer._action_send_sms()
         self.assertSMSNotification([{'number': self.random_numbers_san[0]}], self._test_body)
 
+    def test_composer_comment_invalid_field(self):
```

Comment:
Tests should be ordered alphabetically so that they match the order of execution

Reply:
Mm, still not ordered alphabetically